src/topojson/topojson_intf.ml

```diff
@@ -241,3 +241,29 @@ module type Topojson = sig
       parser and constructor. *)
   module Make (J : Json) : S with type json = J.t
 end
+
+module type Accessor = sig
+  type t
+
+  module Geometry : Geometry with type t = t
+
+  val get_point :
+    Geometry.geometry -> (Geometry.Point.t, [> `Msg of string ]) result
+
+  val get_point_exn : Geometry.geometry -> Geometry.Point.t
+
+  val get_linestring :
+    Geometry.geometry -> (Geometry.LineString.t, [> `Msg of string ]) result
+
+  val get_linestring_exn : Geometry.geometry -> Geometry.LineString.t
+
+  val get_polygon :
+    Geometry.geometry -> (Geometry.Polygon.t, [> `Msg of string ]) result
+
+  val get_polygon_exn : Geometry.geometry -> Geometry.Polygon.t
+
+  val get_multipolygon :
```

Comment:
Looking good, let's add _all_ of the geometries! :))

Reply:
@patricoferris Okay